libvast/vast/table_slice_column.hpp

```diff
@@ -0,0 +1,58 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#pragma once
+
+#include "vast/detail/operators.hpp"
+#include "vast/fwd.hpp"
+#include "vast/table_slice.hpp"
+#include "vast/type.hpp"
+
+#include <tuple>
+#include <vector>
+
+namespace vast {
+
+namespace system {
+
+class index_state;
+class partition;
+using partition_ptr = std::unique_ptr<partition>;
+
+} // namespace system
+
+struct table_slice_column {
+  table_slice_column() {
+  }
+
+  table_slice_column(table_slice_ptr slice_, size_t column_)
+    : slice{std::move(slice_)}, column{column_} {
```

Comment:
I would prefer not to use `_` as suffix only for member variables.

```suggestion
  table_slice_column(table_slice_ptr slice, size_t col)
    : slice{std::move(slice)}, column{col} {
```

Does this trigger a warning by your compiler? For me it just works.

Reply:
It does work for me as well. I'm bit surprised.